admin/c2cgeoportal_admin/views/layertree.py

```diff
@@ -54,6 +53,7 @@ def render_node(nodes, item, parent_id=None, path=''):
                 'id': item.id,
                 'item_type': item.item_type,
                 'name': item.name,
+                'translated_name': client_translation_string_factory(item.name),
```

Comment:
We discussed that we will call request.localizer here to avoid including client domain in _

Reply:
@arnaud-morvan I don't understand, for me this is correct...

Reply:
This return a TranslationString object, not a translated text. The tranlations needs to be done by request.localizer. Calling localizer here with the good domain will avoid to add client domain in auto_translate.

Reply:
+1 @arnaud-morvan => something like:
```
request.localizer.translate(client_translation_string_factory(item.name))
```